src/content/docs/es/guides/endpoints.mdx

```diff
@@ -95,10 +95,12 @@ Todo lo descrito en la sección de endpoints de archivos estáticos también se
 
 Pero, a diferencia del modo `static`, cuando configuras el modo `server`, los endpoints se construirán cuando se soliciten. Esto desbloquea nuevas funciones que no están disponibles al momento de la compilación y permite crear rutas de API que escuchan solicitudes y ejecutan código de forma segura en el servidor en tiempo de ejecución.
 
+Tus rutas se renderizarán bajo demanda de forma predeterminada en el modo `server`. En el modo `hybrid`, debes optar por no pre-renderizar para cada endpoint personalizado con `export const prerender = false`.
+
 <RecipeLinks slugs={["es/recipes/call-endpoints"]}/>
 
 :::note
-Asegúrate de [habilitar SSR](/es/guides/server-side-rendering/) antes de probar estos ejemplos.
+Asegúrate de [habilitar el modo de renderizado bajo demanda](/es/guides/server-side-rendering/) antes de probar estos ejemplos y optar por el prerenderizado en modo `hybrid`. 
```

Comment:
I believe there's a small error here. In the original there's:
> and opt out of prerendering in `hybrid` mode.

But here it looks like it is asking you for opt *into* prerendering on hybrid mode (vs. opt out), which I think should be translated as "optar por no"